src/main/java/com/techsophy/tsf/runtime/form/utils/TokenUtils.java

```diff
@@ -182,4 +182,23 @@ public Sort getSortBy(String[] sortByArray)
         }
         return sort;
     }
+    public static Optional<String> getTenantName()
```

Comment:
why is this method added, awgment-commons should already have it.

Reply:
no sir for userName we have it for tenant we doesnt have

Reply:
not sure how is tenant context being set, without extracting the tenant from token, there must be some reusable method already present.